src/plugin.ts

```diff
@@ -21,11 +21,11 @@ export function buildEnd(err?: Error): void {
 }
 
 function useStyleSheetTransform(filename: string) {
-  return filename.match(/\.s?(a|c)ss/);
```

Comment:
This matches files with extensions like `.css.js`. e.g. https://github.com/adobe/spectrum-web-components/blob/main/packages/overlay/src/OverlayTrigger.ts#L39